### ISSUE TITLE
More naming

### ISSUE THREAD

**Issue by SavageWilliam** [FACN1/week4-FACNhosers#51]
Your front end JS files are quite hard to read, mostly due to naming. Just to highlight a few areas...

- ''/makeRequest" - this is quite a vague name for a route

- one line in main.js is pretty cryptic. Good that you're using a modular design, but the main.js file only relies on functions from a single file... so why not invoke it there?

- redundant 'function eventListeners' ?

- in request.js looking at the function definition.. I'd have no idea 
```callback(response);``` is a function that renders the result, without tracking back to find where tis called

**Comment by m4v15** [comment]
I agree with a lot of stuff - yeah  the route is weird and eventListeners is rendundent now and one line is quite cryptic

But in request.js - we've defined request to call the callback function on the response. Maybe at some in the future we wouldn't want it to render, maybe we will make an API request with the result, so we kind of designed it this way. It doesn't render the result until we call it with the render function as the callback, so it shouldn't say render when we define the request function.
